client/src/components/Search/Search.tsx

```diff
@@ -68,6 +70,7 @@ const Search = ({ search, handleChange }: Props): JSX.Element => {
         getOptionLabel={(option) => option.username}
         options={options}
         loading={loading}
+        onChange={(e, value: any) => value && value.id && setFriendId(value.id)}
```

Comment:
Can we use the correct type for value here? :)

Reply:
I tried about 50 different things and asked Brian to look at this. Unfortunately nothing we tried provided a solution. However, this value comes from a database response that is checking the data type before it gets to this point. The issue here is that the Autocomplete API from Material-UI takes the object, but also adds optional "string" or "null" values to it. So the id could not be guaranteed. I tried everything to get around this.

Reply:
Nevermind. Crushed it! Haha. I think it's ready to go now.